AberrantSMPP/AberrantSMPP.csproj

```diff
@@ -44,8 +45,8 @@
     <AssemblyOriginatorKeyFile>AberrantSMPP.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
May you upgrade this to a current Common.Logging version (supporting net45)?

Reply:
In the actual commit, I have updated the Common.Logging version to v3.4.1